<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="tb_var"/>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>连接设置</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>调试器</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_interface">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>目标芯片</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_target">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bt_conn">
         <property name="text">
          <string>连接目标</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bt_reset">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>复位并运行</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>工程设置</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>符号文件(AXF/ELF)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="txt_axf_path">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>请设置符号文件</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bt_set_axf">
         <property name="text">
          <string>设置符号文件</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>893</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="action_refresh_conf"/>
    <addaction name="separator"/>
    <addaction name="action_save"/>
    <addaction name="action_load"/>
    <addaction name="separator"/>
    <addaction name="action_export"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_help"/>
    <addaction name="separator"/>
    <addaction name="action_checkupdate"/>
    <addaction name="action_homepage"/>
    <addaction name="separator"/>
    <addaction name="action_feedback"/>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>窗口</string>
    </property>
    <addaction name="action_show_graph"/>
    <addaction name="action_show_selector"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_save">
   <property name="text">
    <string>保存配置</string>
   </property>
  </action>
  <action name="action_load">
   <property name="text">
    <string>导入配置</string>
   </property>
  </action>
  <action name="action_export">
   <property name="text">
    <string>导出数据</string>
   </property>
  </action>
  <action name="action_help">
   <property name="text">
    <string>使用说明</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_show_graph">
   <property name="text">
    <string>显示绘图窗口</string>
   </property>
  </action>
  <action name="action_refresh_conf">
   <property name="text">
    <string>刷新连接配置</string>
   </property>
   <property name="toolTip">
    <string>刷新连接配置</string>
   </property>
  </action>
  <action name="action_homepage">
   <property name="text">
    <string>转到主页</string>
   </property>
  </action>
  <action name="action_show_selector">
   <property name="text">
    <string>显示选择窗口</string>
   </property>
  </action>
  <action name="action_feedback">
   <property name="text">
    <string>反馈</string>
   </property>
  </action>
  <action name="action_checkupdate">
   <property name="text">
    <string>检查更新</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
